packages/base/Avatar/src/Avatar/Avatar.tsx

```diff
@@ -126,7 +119,7 @@ export const Avatar = (props: Props) => {
 
   if (isEditable) {
     return (
-      <div className={classes.editableAvatarContainer}>
+      <div className={'relative'}>
```

Comment:
```suggestion
      <div className='relative'>
```
We try to write all string-only props like this. I actually thought we have an eslint/prettier setup for this but it looks like we don't 🤔